Library/Homebrew/extend/os/mac/update-report.rb

```diff
@@ -0,0 +1,12 @@
+# typed: false
```

Comment:
Calling this `os/mac/cmd/update-report` may be better (and same for below file too)